commands/fun/cat.js

```diff
@@ -13,10 +13,11 @@ exports.exec = async (Bastion, message) => {
       url: 'https://thecatapi.com/api/images/get',
       followAllRedirects: true
     };
-    let response = await request(options);
+    let pic;
+    await request(options, function (error, response) { pic = response.request.uri.href; });
```

Comment:
The whole point of using a `promise` based `request` library is to avoid using callbacks.

If you want access to the `request` object in the resolved value of the function. You can simply set the `resolveWithFullResponse` to `true`.

For example,
```js
let options = {
      method: 'HEAD',
      url: 'https://thecatapi.com/api/images/get',
      resolveWithFullResponse: true,
      followAllRedirects: true
};
let response = await request(options);

/*
 * `response.request.href` gives the URL for the cat's image. Use this to upload the
 * cat's image through Bastion
 */
console.log(response.request.href)
```